mailpoet/assets/js/src/automation/integrations/woocommerce/steps/buys-from-a-tag/edit/tag-autocompleter.tsx

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
Mostly taken from the categories autocompleter in WooCommerce here: https://github.com/woocommerce/woocommerce-admin/blob/78d0e10a99676c92650a5248917207e7304f5c1b/packages/components/src/search/autocompleters/categories.js